docs/sharding.md

```diff
@@ -50,6 +50,11 @@ You can add more replica sets with different names to the `replsets` section
 in a similar way. Please take into account that having more than one replica set
 is possible only with the sharding turned on.
 
+!!! note
+
+    To remove a shard with the Operator you must first delete all application
```

Comment:
just delete seems a little bit harsh, I think we should also mention possibility of moving non sharded collections from this shard (with `movePrimary`) to the shard which is not deleted